### ISSUE TITLE
jsonschema segmentation fault

### ISSUE THREAD

**Issue by kmilos** [msys2/MINGW-packages#11463]
I guess happened after one of the recent python-jsonschema updates...

MINGW64 produces "Segmentation fault", while CLANG64 "Fatal error in launcher: Unable to find an appended archive."

**Comment by kmilos** [comment]
I think stripping was on by default before the update? Could be this this commit maybe? @lazka 

https://github.com/msys2/MINGW-packages/commit/8ce84a6c0efdc3569d536b2dc2ff44cca4c41bfd

**Comment by kmilos** [comment]
> Those new launchers

Or you mean the launcher changed since switching away from setuptools as well?

**Comment by jeremyd2019** [comment]
> Or you mean the launcher changed since switching away from setuptools as well?

Yes, the setuptools launchers are different from what I guess I'll call the distlib launchers (they get bundled into distlib, pip, and python-installer at least)

**Comment by jeremyd2019** [comment]
Yes, that's it, dd0f45fe3969b50bb5dff6e7819f85e38dacdd9b switched from setuptools to python-installer but didn't disable stripping.

**Comment by kmilos** [comment]
Thanks, let's give that a go.

**Comment by Biswa96** [comment]
> switched from setuptools to python-installer but didn't disable stripping.

Should stripping be disabled for every python packages which was built with python-installer?

**Comment by Biswa96** [comment]
Out of curiosity, would you like to explain the reason please? Any docs or code would help also.

**Comment by kmilos** [comment]
FYI, #11466 works indeed!
